bar.go

```diff
@@ -1,22 +1,94 @@
 package desktop
 
 import (
+	"fmt"
+	"io/ioutil"
 	"os/exec"
+	"path/filepath"
+	"strings"
 
 	"fyne.io/fyne"
 	"fyne.io/fyne/layout"
-	"fyne.io/fyne/widget"
 )
 
-func newBar() fyne.CanvasObject {
-	return fyne.NewContainerWithLayout(layout.NewHBoxLayout(),
-		layout.NewSpacer(),
-		widget.NewButton("Browser", func() {
-			exec.Command("chromium").Start()
-		}),
-		widget.NewButton("Terminal", func() {
-			exec.Command("xterm").Start()
-		}),
-		layout.NewSpacer(),
-	)
+var appBar *Bar
+var fyconSize = 32
+var fyconTheme = "Papirus"
```

Comment:
Hicolor is already hard coded in as the fallback - but I can change this to hicolor for now... It doesn't matter as much because I plan to add a settings panel for this as soon as this pr is accepted.